Detectors/MUON/MID/Workflow/src/RawDecoderSpec.cxx

```diff
@@ -59,6 +59,7 @@ class RawDecoderDeviceDPL
     }
 
     mDecoder->clear();
+    o2::header::DataHeader const* dh = nullptr;
```

Comment:
this line is not needed, is it ?

Reply:
Indeed it is not needed anymore: thanks for spotting this!